### ISSUE TITLE
PG-5 PcGun by A4 Tech Support?

### ISSUE THREAD

**Issue by phasermaniac83** [schellingb/dosbox-pure#327]
There are two obscure Dos games that used lightguns back in the day, moreover they supported 2p action:
AD cop
Bonbon paradise
In both games you have a Dossteup.exe where you can set your controller as lightgun but as espected as it's very rare, it won't work.
If you set controller to mouse you can play with modern lightguns, but only one player allowed.
In windows, nowadays lightguns like Aimtraks or Sinden or even Crt ones, are seen as absolute coord mice, and uses rawinput to allow two pointers at once.
https://www.youtube.com/watch?v=PKdaGMO1QQo
In this video he talks about some technical details.

**Comment by Tovarichtch** [comment]
I would love to see the lightgun support too.

**Comment by Darknior** [comment]
Good idea, for sure we can play Gun games with mouse. But with mouse there is only one player available :(
A real Gun support would be welcome
